packages/components/src/components/resource-link/src/ResourceAnchor.tsx

```diff
@@ -1,47 +1,44 @@
 import * as React from "react";
+import { styled } from "@trueplan/forecast-theme";
 import type { AnchorProps } from "../../anchor";
 import { Anchor } from "../../anchor";
 
 export type ResourceAnchorProps = AnchorProps;
 
+const StyledAnchor = styled(Anchor, {
+  position: "relative",
+  width: "100%",
+  height: "100%",
+  "&:hover": {
+    border: "none",
+  },
+  "&:focus": {
+    border: "none",
+  },
+  "&:after": {
+    content: "",
+    position: "absolute",
+    zIndex: -1,
+    width: "100%",
+    height: "100%",
+    opacity: 0,
+    top: 0,
+    left: 0,
+    borderRadius: "$30",
+    boxShadow: "0 5px 15px rgba(60, 72, 88,0.3)",
+    transition: "opacity 0.3s ease-in-out",
+  },
+  "&:hover:after": {
+    opacity: "1",
+  },
+});
+
 const ResourceAnchor = React.forwardRef<HTMLAnchorElement, ResourceAnchorProps>(
   ({ children, href, ...props }, ref) => {
     return (
-      <Anchor
-        css={{
-          position: "relative",
-          width: "100%",
-          height: "100%",
-          "&:hover": {
-            border: "none",
-          },
-          "&:focus": {
-            border: "none",
-          },
-          "&:after": {
-            content: "",
-            position: "absolute",
-            zIndex: -1,
-            width: "100%",
-            height: "100%",
-            opacity: 0,
-            top: 0,
-            left: 0,
-            borderRadius: "$30",
-            boxShadow: "0 5px 15px rgba(60, 72, 88,0.3)",
-            transition: "opacity 0.3s ease-in-out",
-          },
-          "&:hover:after": {
-            opacity: "1",
-          },
-        }}
-        href={href}
-        noUnderline
-        {...props}
-        ref={ref}
-      >
+      <StyledAnchor href={href} noUnderline {...props} ref={ref}>
```

Comment:
Or we can use the `styled` function to override/extend existing component styles.